README.md

```diff
@@ -1,242 +1,278 @@
+  quint_team_secret_santa::secret_santa::recipient_for_santa: Map()
+}
 
-## Name origin
+[State 1]
+{
+  quint_team_secret_santa::secret_santa::bowl: Set("Igor", "Jure", "Shon", "Thomas"),
+  quint_team_secret_santa::secret_santa::recipient_for_santa: Map("Gabriela" -> "Gabriela")
+}
 
-Quint is short for Quintessence, from alchemy, which refers to the fifth
-element. A lot of alchemy is about transmutation and energy, and Quint makes it
-possible to transmute specifications into executable assets and empower ideas to
-become referenced artifacts.
+[violation] Found an issue (2047ms).
+error: found a counterexample
+```
 
-## Documentation
+</details>
 
-### Tutorials :teacher:
+[Apalache]: https://github.com/informalsystems/apalache
+[TLA]: https://en.wikipedia.org/wiki/Temporal_logic_of_actions
 
-Visit the [Tutorials][] page.
+### Features
+<dl>
+  <dt>A simple and familiar <strong>syntax</strong></dt>
+  <dd>to support engineers reading and writing specifications</dd>
 
-### Syntax :abcd:
+  <dt>An expressive <strong>type systems</strong></dt>
+  <dd>to ensure the domain model is coherent</dd>
 
-- [Cheatsheet](./doc/quint-cheatsheet.pdf)
-- [Reference API documentation for built-in operators](./doc/builtin.md)
-- [Syntax documentation](./doc/lang.md)
-- [Frequently asked questions](./doc/faq.md)
+  <dt>A novel <strong>effect system</strong></dt>
+  <dd>to ensure state updates are coherent</dd>
 
-### Examples :musical_score:
+  <dt><strong>IDE support</strong> via LSP</dt>
+  <dd>giving real time feedback when writing specifications</dd>
 
-We have written [examples](./examples) of several specifications in Quint.
-Some of them accompany a TLA+ version for comparison and learning purposes.
-To simplify reading, use [syntax highlighting](./editor-plugins) for your
-editor (currently, VSCode, Emacs and Vim are supported).
+  <dt>A <strong>REPL</strong></dt>
+  <dd>enabling interactive exploration of specifications</dd>
 
-## Community and help
+  <dt>A <strong>simulator</strong></dt>
+  <dd>enabling tests, trace generation, and exploration of your system</dd>
 
-- Join the chat in the [Quint zulip stream][] :telephone:
-- Join the [Quint discussions][] :bulb:
-- [Contribute your spell][] to the collection of Quint spells :scroll:
-- [Contribute](./CONTRIBUTING.md) to the development of Quint :construction_worker:
+  <dt><strong>Symbolic model checking</strong></dt>
+  <dd>to verify your specifications via <a href="https://github.com/informalsystems/apalache">Apalache</a></dd>
+</dl>
 
-### Tools :space_invader:
+### Motivation
 
-[Quick previews](./doc/previews.md) of the tools.
+Quint is inspired by [TLA+][] (the language) but provides an alternative surface
+syntax for specifying systems in TLA (the logic). The most important feature of
+our syntax is that it is minimal and regular, making Quint an easy target for
+advanced developer tooling and static analysis (see our [design principles][]
```

Comment:
Looks like the link target here is missing for `[design principles]`.

Reply:
Fixed. Thanks!